_LowLevelCode/cpp/cpp_communicator/cpp_communicator.cpp

```diff
@@ -190,10 +199,10 @@ void mexFunction(int nlhs, mxArray* plhs[], int nrhs, const mxArray* prhs[])
 /* If appropriate number of output arguments are availiable, set up the mex routine output arguments to mpi_numLab and mpi_labNum values
    extracted from initialized MPI framework.
 */
-void set_numlab_and_nlabs(class_handle<MPI_wrapper> const* const pCommunicatorHolder, int nlhs, mxArray* plhs[], int nrhs, const mxArray* prhs[]) {
+void set_numlab_and_nlabs(class_handle<MPI_wrapper> * const pCommunicatorHolder, int nlhs, mxArray* plhs[], int nrhs, const mxArray* prhs[]) {
 
     if (nlhs >= (int)labIndex_Out::numLab + 1) {
-        plhs[(int)labIndex_Out::numLab] = mxCreateNumericMatrix(1, 1, mxUINT64_CLASS, mxREAL);
+        plhs[(int)labIndex_Out::numLab] = mxCreateNumericMatrix(1, 1, mxINT32_CLASS, mxREAL);
```

Comment:
why switch from `UINT64` to `INT32` here?

Reply:
because inputs become 32 -- I think the number of parallel workers  of  2^31 order is well sufficient for any imaginable purpose